src/db/migration/db_migration.rs

```diff
@@ -79,6 +80,10 @@ impl DbMigration {
     }
 }
 
+pub(crate) fn db_name(from: &Version, to: &Version) -> String {
```

Comment:
Plus, I think it belongs more in the `migration_map.rs` file.

Reply:
Since it's used to implement `db_migration` trait's method - I don't see a reason to move it out of the file. But if needed - we can do it. No strong opinion here.

Reply:
It's used to implement `MigrationOperation` trait methods defined in `migration_map.rs`, no?